src/Jackett.Common/Definitions/xbytes2.yml

```diff
@@ -57,6 +59,21 @@
             args: category
       title:
         selector: td[valign="middle"] a
+        filters:
+          - name: re_replace
+            args: ["S0(\\d{1,2})\\/E(\\d{1,2})", "$1x$2"]
```

Comment:
This probably will break other applications. Why are you rewriting S0XX/EYY to XXxYY?
Just stripping the / should be enough?!?